### PR TITLE
(docs) update bind_paths_and_mounts.rst

### PR DESCRIPTION
**Pull request by vladdoster**
## Description of the Pull Request (PR):

- spelling
- reduce verbiage
- fix punctuation

## This fixes or addresses the following GitHub issues:

N/A

### CODE REVIEW

File: bind_paths_and_mounts.rst [hunk 1]
```diff
@@ -105,7 +105,7 @@ You can bind multiple directories in a single command with this syntax:
 This will bind ``/opt`` on the host to ``/opt`` in the container and ``/data``
 on the host to ``/mnt`` in the container.
 
-Using the environment variable instead of the command line argument, this would
+Using the environment variable instead of the command line argument, would
```
Comment (dtrudg) [inline bind_paths_and_mounts.rst:108]: The `this` should not be removed.

File: bind_paths_and_mounts.rst [hunk 2]
```diff
@@ -276,9 +276,9 @@ type:
 Image Mounts
 ------------
 
-In Singularity 3.6 and above you can mount a directory contained in an
+In Singularity 3.6 and above, you can mount a directory contained in an
```
Comment (dtrudg) [inline bind_paths_and_mounts.rst:279]: comma shouldn't be added, the statement is directly about Singualrity 3.6 and above.

File: bind_paths_and_mounts.rst [hunk 3]
```diff
@@ -340,7 +340,7 @@ SquashFS Image Files
 ====================
 
 If you have a directory called ``inputs/`` that holds data files you wish
-to distribute in an image file that is read-only, and compressed, then
+to distribute in a read-only image file, and compressed, then
```
Comment (dtrudg) [inline bind_paths_and_mounts.rst:343]: Please leave original text, which specifies the image file itself is read-only and compressed.

Suggested change is not specific that the image file provides the compression.

File: bind_paths_and_mounts.rst [hunk 4]
```diff
@@ -380,7 +380,7 @@ overlays instructions<overlay-sif>`:
     Singularity> ls /input-data
     1  2  3  4  5  6  7  8  9
 
-If your bind source is a SIF then Singularity will bind from
-the first data partition in the SIF, or you may specify an
+If your bind source is a SIF, then Singularity will bind from
```
Comment (dtrudg) [inline bind_paths_and_mounts.rst:383]: Comma not needed here. Better overall would be split into to sentences without adding a comma.

> If your bind source is a SIF then Singularity will bind from the first data partition in the SIF. You may specify an alternative descriptor by ID with the additional bind option ...

**Review by dtrudg — CHANGES_REQUESTED** [review]

**Comment by brianclemens** [comment]
Closed for inactivity.

**Pull request closed without merging**